### ISSUE TITLE
The legacy REST API endpoint (https://transparency.entsoe.eu/api) has been discontinued

### ISSUE THREAD

**Issue by CesarPereiraa** [EnergieID/entsoe-py#241]
Hi, I have been using this package for a while now,using always EntsoePandasClient.
To my surprise, today I tried to run the following code and got an error.
code:
![code](https://user-images.githubusercontent.com/88057602/219097855-8e47a839-04bb-46e3-be54-f84a26a6ad3c.PNG)

with the following error:
![error](https://user-images.githubusercontent.com/88057602/219095571-0c474373-7257-4527-9ad5-7239d46d5788.png)

When following the url and checking the entsoe page I saw the following statement:
![entose](https://user-images.githubusercontent.com/88057602/219096280-8bd7cc8d-635d-4915-982d-2aa3166f4c5f.PNG)

I would like to ask if it would be possible to make the change to this new API in order to continue using this package and be able to directly remove the data in Pandas Series or DataFrame?

Thanks in advance!

**Comment by fboerman** [comment]
Hi this has been fixed months ago in a few relaeases back. Please update the package. I would like to kindly request you to test things with the latest version of the package first before opening an issue.

-------- Original Message --------

**Comment by CesarPereiraa** [comment]
Hi fboerman,
Thanks for the quick response! The first thing I did was update the package to the latest version and it was still returning the same error. I found your answer strange and restart the Jupyter Notebook and the code has already run...

I want to apologize for the inconvenience and embarrassment of opening an issue for such a stupid mistake...

I want to thank you again for the answer and I will delete this subject later.
